### ISSUE TITLE
Remember search options preferences

### ISSUE THREAD

**Issue by VulpesFerrilata** [Dorset-Council-UK/GIFramework-Maps#13]
User's can update their search preferences by selecting the cog next to the search box:

![image](https://user-images.githubusercontent.com/114388582/214597135-63ea5b84-ae5e-4b64-b8f7-bccb40357b0c.png)

However, these preferences are lost if the page is refreshed (default is show in the screenshot below - with everything ticked). We would like the search preferences to be remembered.

![image](https://user-images.githubusercontent.com/114388582/214596780-40b70516-e9b3-4701-aa51-f49b620649fb.png)

**Comment by paul-dorsetcouncil** [comment]
@RobQuincey was this ticket resolved in your last pull request? If so, can we close this one?

**Comment by RobQuincey** [comment]
No, this bit wasn't 'remembered', it was just the single option that we added for multiple search results that is remembered, not the full list, so this needs to remain open

**Comment by paul-dorsetcouncil** [comment]
Ah right, thank you for confirming :-)
